python/cmdLineUtils/CMakeLists.txt

```diff
@@ -348,6 +354,19 @@ ROOTTEST_ADD_TEST(SimpleRootmv5CheckOutput
 ROOTTEST_ADD_TEST(SimpleRootmv5Clean
                   COMMAND ${TOOLS_PREFIX}/rootrm -r move5.root
                   DEPENDS SimpleRootmv5CheckOutput)
+
+
+ROOTTEST_ADD_TEST(NameCyclesRootmvPrepareInput
+                  COMMAND root -q -l MakeNameCyclesRootmvInput.C)
```

Comment:
Typically we use
```
root.exe -b -l -q
```
to save a bit of time :)

Reply:
fixed, thanks!